python_training/sim_city.py

```diff
@@ -0,0 +1,198 @@
+BASE_MUNICIPALITY_TAX_RATE = 1000
+
+
+class Ackland:
+    """
+    The state Ackland itself.
+    """
+
+    def __init__(self, cities):
+        """
+        Initiates the Ackland state object.
+        """
+        if type(cities) is City:
```

Comment:
Better to use `isinstance` for type checking in Python:
`if isinstance(cities, City):`
And, a typing hint would be nice here (and generally in c'tors).

Reply:
Im not sure i understood what you'v meant, show me tomorrow :)